neo/renderer/tr_light.cpp

```diff
@@ -1062,9 +1062,12 @@ R_IssueEntityDefCallback
 */
 bool R_IssueEntityDefCallback( idRenderEntityLocal *def ) {
 	bool update;
-	idBounds	oldBounds;
+	idBounds oldBounds;
 	const bool checkBounds = r_checkBounds.GetBool();
 
+	// Initialize oldBounds to avoid the uninitialized variable warning
```

Comment:
it's not used uninitialized, it's only used `if ( checkBounds )`, and in that case it's initialized